frontend/src/theme/textStyles.tsx

```diff
@@ -0,0 +1,31 @@
+const font = "DM Sans"
+
+const textStyles = {
+  "body-medium": {
+    fontFamily: font,
+    fontSize: "16px",
+  },
+  "header-medium": {
+    fontFamily: font,
+    fontWeight: 400,
+    fontSize: "16px",
+  },
+  "header-large": {
+    fontFamily: font,
+    fontWeight: 500,
+    fontSize: "28px",
+  },
+  "button-medium": {
+    fontFamily: font,
+    fontWeight: 400,
+    fontSize: "18px",
+  },
+  "button-navbar": {
+    fontFamily: font,
+    fontSize: "16px !important",
```

Comment:
not a big fan of using !important... overwrites scare me and can cause issues later. Is there a better workaround for this?

Reply:
you actually don't need the important flag here
```suggestion
    fontSize: "16px",
```